content/tutorial/tutorial.md

```diff
@@ -956,7 +963,7 @@ Since the Game component is now rendering the game's status, we can remove the c
   }
 ```
 
-Finally, we need to move the `handleClick` method from the Board component to the Game component. We also need to modify `handleClick` because the Game component's state is structured differently. Within the Game's `handleClick` method, we concatenate new history entries onto `history`.
+अंत में, हमें Board कौम्पोनॅन्ट से Game कौम्पोनॅन्ट में `handleClick` मेथड को स्थानांतरित करने की आवश्यकता है। हमें `handleClick` को भी संशोधित करने की आवश्यकता है क्योंकि गेम घटक की स्थिति अलग तरह से संरचित है। गेम के `handleClick` पद्धति के तहत, हम `history` पर नए इतिहास प्रविष्टियों को बदलते हैं।
```

Comment:
स्थानांतरित -> too complex?

Reply:
revisit once

Reply:
```
अंत में, हमें Board कौम्पोनॅन्ट से Game कौम्पोनॅन्ट में `handleClick` मेथड को बदलने करने की आवश्यकता है। हमें `handleClick` को भी संशोधित करने की आवश्यकता है क्योंकि Game कौम्पोनॅन्ट की स्थिति अलग तरह से संरचित है। गेम के `handleClick` पद्धति के तहत, हम `history` पर नए इतिहास प्रविष्टियों को बदलते हैं।
```

i didn't see any change in this other than couple of words. Any suggestions on this?

Reply:
संशोधित -> badalne and अलग तरह से संरचित है -> alag tarah se banai gai hai?

गेम के `handleClick` पद्धति के तहत, हम `history` पर नए इतिहास प्रविष्टियों को बदलते हैं। is still complex

Reply:
> संशोधित -> badalne and अलग तरह से संरचित है -> alag tarah se banai gai hai?
> 
I have changed it to this
```
हमें `handleClick` को भी बदलने और अलग तरह से बनाने की आवश्यकता है
```

Reply:
> गेम के `handleClick` पद्धति के तहत, हम `history` पर नए इतिहास प्रविष्टियों को बदलते हैं। is still complex

```
गेम के `handleClick` मेथड के अंदर हम नयी `history` को पहले की `history` के साथ जोड़ते हैं 
```

how about this?

Reply:
घटक should be component in hindi

Reply:
की स्थिति अलग तरह से संरचित है। -> की state अलग तरह से sthapit ki gai hai?

Reply:
पद्धति -> method in hindi?

Reply:
प्रविष्टियों -> entries in hindi? it is too complex otherwise